Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM debian:buster-slim
+LABEL maintainer "Hiroki Kumazaki <kumagi@google.com>"
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends -q \
+    wget \
+    unzip \
+    perl \
+    make \
+    clang \
+    gcc \
+    git && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN wget --no-check-certificate https://github.com/Kitware/CMake/releases/download/v3.15.2/cmake-3.15.2-Linux-x86_64.tar.gz && \
+     tar -xf cmake-3.15.2-Linux-x86_64.tar.gz
+ENV PATH $PATH:/cmake-3.15.2-Linux-x86_64/bin
+
+RUN wget --no-check-certificate https://github.com/openssl/openssl/archive/OpenSSL_1_1_1c.zip && \
```

Comment:
Would `apt-get install ca-certificates` instead of disabling cert check work?
It seems that certs (packed into the docker image) are outdated and need update.
[Random Q/A from stadckoverflow](https://stackoverflow.com/questions/37970990/ssl-certificate-verification-fails-inside-docker-container-on-specific-server)

Also I'm wondering whether cmake-3.15.2 is really required to build instead of the one in apt.

Reply:
OK, cmake and openssl is good with debian-buster's default package versions.
So, `wget` commands are eliminated, then `ca-certificates` are unneeded.